### ISSUE TITLE
Extra Debloating

### ISSUE THREAD

**Issue by EmperorCrusher** [Raphire/Win11Debloat#143]
@Raphire, I use Win11Debloat on all my new PC installs. One thing missing was the ability to debloat Office 365. I personally keep the English version, but debloat the Spanish and French. This also includes removing OneNote. Since I sell Dell PCs to my clients, I also remove a lot of Dell bloatware. I wound up writing a script to facilitate this based on your scripting methods and a LOT of research. If there is any interest in adding any of this to your script, ping me and I'll forward it to you.

I've created another script to do more debloating I have not seen here and can forward it to you as well if you are interested. It includes:

- Removing Microsoft Store from the taskbar (not uninstalling the store, just removing from the taskbar)
- Turn off OneDrive in Settings > Apps > Startup (not removing OneDrive completely)

My script does more, but it's less debloating and more stopping annoyances like making Windows stop asking to make changes after some critical or feature updates. Silence is golden, MS. LMK.

**Comment by johnnyq** [comment]
Super excited for this Please Share, Submit a Pull Request

**Comment by EmperorCrusher** [comment]
I'm not at all experienced at submitting things at Github. I can try following my nose to see if it winds up at the right place or can someone explain how to do this?

**Comment by Raphire** [comment]
Sounds good. I'm always open to adding more options to the script.

> I'm not at all experienced at submitting things at Github. I can try following my nose to see if it winds up at the right place or can someone explain how to do this?

If you fork the repo you can make the changes on your fork and later create a pull request [here](https://github.com/Raphire/Win11Debloat/pulls).

**Comment by OmaigatTER** [comment]
It should be a auto-detection to see if te user uses the app and if it does don't remove it

**Comment by EmperorCrusher** [comment]
I would agree with @OmaigatTER that if this script was to be run on a system **in production**, that it would be best to see if the user actually uses the app. I don't know how to do that, so I'll leave this up to more educated developers like @OmaigatTER to contribute code to help. Would you mind helping?

Also, to help everyone understand why this script even exists, I'm a solo IT provider and purchase dozens of PCs/laptops from Dell for my clients. I spent a lot of time doing the debloating of Dell and MS products ON BRAND NEW PCs being pushed out to my clients, so I came up with this script. This works best for me pushing out BRAND NEW PCs for clients that I know and take care of. It will not work well for everyone and certainly would upset some who use some of the software on systems already in production. But I know my client base and they consider these things clutter, so I'd be comfortable running it on my own client's production systems. As @OmaigatTER pointed out, this will NOT be the case for everyone.

**Comment by Raphire** [comment]
Closing this as the individual issues are described in #343 and #136 respectively.
